tools/packaging/static-build/ovmf/build.sh

```diff
@@ -15,7 +15,7 @@ source "${script_dir}/../../scripts/lib.sh"
 
 DESTDIR=${DESTDIR:-${PWD}}
 PREFIX=${PREFIX:-/opt/kata}
-container_image="${OVMF_CONTAINER_BUILDER:-$(get_ovmf_image_name)}"
+container_image="${OVMF_CONTAINER_BUILDER:-${BUILDER_REGISTRY}:ovmf-$(get_last_modification ${script_dir})-$(uname -m)}"
```

Comment:
I think this needs reverting to the old approach to pick up `CC_BUILDER_REGISTRY` from lib.sh rather than ```suggestion
container_image="${OVMF_CONTAINER_BUILDER:-$(get_ovmf_image_name)}"
```